lib/verdict/experiment.rb

```diff
@@ -136,7 +147,7 @@ def assign(subject, context = nil)
     subject_identifier = retrieve_subject_identifier(subject)
     assignment = if previous_assignment
                    previous_assignment
-                 elsif subject_qualifies?(subject, context)
+                 elsif subject_qualifies?(subject, context) && (not is_stop_new_assignments?)
```

Comment:
nit: Since this is the only place we are calling the function, we can invert the logic to not need the negation i.e. `is_make_new_assignments?`